src/com/company/Main.java

```diff
@@ -0,0 +1,11 @@
+package com.company;
+
+public class Main
+{
+
+    public static void main(String[] args)
```

Comment:
![image](https://user-images.githubusercontent.com/67580205/161382063-bb51d1d1-2182-4ea4-af8f-b8ffc79bb655.png)
[Fix] daca vreau sa printez materiile, si nu e niciuna, ramane hanging fara sa-mi zica ce s-a intamplat sau ce ar trebui sa fac
![image](https://user-images.githubusercontent.com/67580205/161382107-3be7b4aa-cdf5-404c-92f9-b5fa59d6fed6.png)
[Fix] daca aleg sa adaug o materie si bag o litera in loc de cifra, crapa. Nu te astepta ca userii sa foloseasca aplciatia ta cum ai vrut tu

Reply:
aplicatia nu simuleaza niciun sistem real, doar adauga si afiseaza